packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -351,12 +351,12 @@ export namespace CommonCommands {
     });
 }
 
-export const supportCut = browser.isNative || document.queryCommandSupported('cut');
-export const supportCopy = browser.isNative || document.queryCommandSupported('copy');
+export const supportCut = environment.electron.is() || document.queryCommandSupported('cut');
```

Comment:
Just to note it down somewhere cause I quickly looked into it: `queryCommandSupported` is officially [deprecated but still supported in all major browsers](https://developer.mozilla.org/en-US/docs/Web/API/Document/queryCommandSupported#browser_compatibility). From what I gather, we could use the [Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) as an alternative.